script/en/validation.php

```diff
@@ -57,8 +52,8 @@
     ],
     'image'                => 'The :attribute must be an image.',
     'in'                   => 'The selected :attribute is invalid.',
-    'in_array'             => 'The :attribute field does not exist in :other.',
     'integer'              => 'The :attribute must be an integer.',
+    'in_array'             => 'The :attribute field does not exist in :other.',
```

Comment:
Your script sorts 'in_array' after 'integer'

Reply:
`ksort($array)`:
```
API key
all key
foo_key
fookey
```

`ksort($array, SORT_FLAG_CASE ^ SORT_NATURAL)`:
```
all key
API key
fookey
foo_key
```

Reply:
I see. I review that above but find the reason here.

Reply:
With the latest changes, the sort looks like this:

![image](https://user-images.githubusercontent.com/10347617/111281253-cc1d2d00-864d-11eb-8c83-cf84c01b41db.png)